libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1472,9 +1474,12 @@ void ExpressionCompiler::appendArithmeticOperatorCode(Token::Value _operator, Ty
 	case Token::Mul:
 		m_context << Instruction::MUL;
 		if (fixedType)
-			m_context << u256(pow(bigint(10), fixedType->fractionalDigits()))
+		{
+			solUnimplemented("Not yet implemented - FixedPointType multiplication");
```

Comment:
Can you enable it at least for the cases where there cannot be any overflow? I.e. where
`fixedFactor < (u256(1) << (numBits / 2))` or something like that?